libheif/heif.cc

```diff
@@ -3685,7 +3685,7 @@ struct heif_error heif_region_get_referenced_mask_ID(const struct heif_region* r
   return heif_error_invalid_parameter_value;
 }
 
-unsigned long int heif_region_get_inline_mask_data_len(const struct heif_region* region)
```

Comment:
Do you think it is safe to change the return type in the public API?
Strictly speaking, it probably is not, but on the other hand, it'll work in most cases and this function is probably currently not in use yet anywhere. (At least that I know of.)

Reply:
There's probably alternative workarounds for this particular instance if you don't want to change the API, which is understandable. OTOH, having platform dependent return types that can catch one out in the API is not ideal either. Since libheif bumps the API and ABI version every once in a while anyway, I'll let you decide if and when you want to do it.

Reply:
So far, we have changed the API only in backwards compatible ways. That would break compatibility.

@fancycode You are more strict in these things than I am. I would be fine to silently change the return type as this function is probably not used in any application code (maybe only in private code of @bradh). But if you would insist in announcing an incompatible API change because of this, I would go for simply casting the return type.

Reply:
> So far, we have changed the API only in backwards compatible ways.

But I think you have changed the ABI in non-backwards compatible ways, so apps required rebuilds anyway in the past?

Reply:
> > So far, we have changed the API only in backwards compatible ways.
> 
> But I think you have changed the ABI in non-backwards compatible ways, so apps required rebuilds anyway in the past?

Not that I remember. Excluding bugs like the `heif_init()` thing that just came up.

Reply:
I'm OK with the change - the work is so developmental, everything is moving.

Reply:
As the public API for `heif_region_get_inline_mask_data_len` was not in a released version yet, I'm fine with changing it.

Reply:
Right.